anonymizer/OpenDiffix.Service/Program.fs

```diff
@@ -134,15 +141,18 @@ let handleExport
     | Rows -> "*"
     | Entities -> $"distinct %s{aidColumn}"
 
+  let countColumn = $"diffix_count(%s{countInput}, %s{aidColumn}) AS count"
```

Comment:
`countColumns` too?

Reply:
I think I prefer the singular: there is only one column here and we don't have plans to add others.

Reply:
I'm blind :see_no_evil: